src/databricks/labs/ucx/cli.py

```diff
@@ -532,6 +535,65 @@ def join_collection(a: AccountClient, workspace_ids: str):
     account_installer.join_collection(w_ids)
 
 
+@ucx.command
+def upload(
+    file: Path | str,
+    w: WorkspaceClient,
+    run_as_collection: bool = False,
+    a: AccountClient | None = None,  # Only used while testing
+):
+    """Upload a file to the (collection of) workspace(s)"""
+    file = Path(file)
+    contexts = _get_workspace_contexts(w, run_as_collection=run_as_collection, a=a)
+    logger.warning("The schema of CSV files is NOT validated, ensure it is correct")
+    for ctx in contexts:
+        ctx.installation.upload(file.name, file.read_bytes())
+    if len(contexts) > 0:
+        logger.info(f"Finished uploading {file}")
+
+
+@ucx.command
+def download(
```

Comment:
I'm a bit skeptical with the concept of merging downloaded files from different workspaces... I suggest downloading from only 1 workspace